### ISSUE TITLE
Exploration of 6-substituted compounds

### ISSUE THREAD

**Issue by maratsydney** [OpenSourceMalaria/Series4#60]
I spend some time exploring tele-substitution phenomenon #46 and as a part of this big study, I intended to check whether 6-chloro starting material introduced to standard conditions of coupling with alcohol/amine/thiol would result in tele-substitution. I checked that we have only one example of 6-substitued compound in our library and it is surprisingly potent (OSM-X-001, IC50=34 nM) compare to 5-substituted analogue (OSM-S-260, IC50=281). This compound is inherited from Pfizer and the reason why we did not follow was a statement from Pfizer that 6-substituted compounds suffer from[ poor metabolic stability](https://github.com/OpenSourceMalaria/Series4/wiki/Modification-of-Pyrazine-Substitution-Pattern). However, looking at [metabolic stability data](https://github.com/OpenSourceMalaria/Series4/wiki/Physicochem-metabolism-PK) available at present moment this statement needs more evidences, since metabolic stability of similar molecule with chlorine on the place of difluoromethoxy group is very similar but we don't have data for OSM-S-260 for direct comparison.

![comparison of 5- vs 6-](https://user-images.githubusercontent.com/29498087/47405414-20819280-d79d-11e8-96fb-714c584da19a.png)

I am currently trying to resynthesise original molecule OSM-X-001 to prove that assay result is actually valid. Then I am planning to explore this area with few more examples, for instance looking at the activity of OSM-S-260 IC50=281 against OSM-S-272 IC50= 38 nM we can expect that molecule that poses 4-chlorophenyl group in position 3 and alcohol residue in position 6 can reach single digits nanomolar activity. Offcourse resulted structure would be far away from desired phys-chem properties but high activity can help us to bypass these limits. On top of this, based on previous results we know that 4-chloro-3-aminophenyl is well tolerated in the position 3 without significant decrease in activity (OSM-S-548, IC50= 45 nM). I believe that it is important to fill in this gap of data for 6-substituted compounds and I would love to hear suggestions in case if I missed something.

![6-substituted suggested structures](https://user-images.githubusercontent.com/29498087/47405422-2aa39100-d79d-11e8-899a-f7ac9c5893c9.png)

I already started synthetic work on these molecules. Good news that synthesis of the core was easy ([MK145-1](https://au-mynotebook.labarchives.com/share/Marat%2520Korsik/NDI2LjQwMDAwMDAwMDAwMDAzfDI5MzcwLzMyOC9UcmVlTm9kZS8xODUyNzUxMTA3fDEwODIuMzk5OTk5OTk5OTk5OQ==)), but coupling reaction is not working in standard condition that we employed in the group before ([MK146-1](https://au-mynotebook.labarchives.com/share/Marat%2520Korsik/NDI3Ljd8MjkzNzAvMzI5L1RyZWVOb2RlLzE4NDE3NzkxMjZ8MTA4NS43)). Reaction goes to violent and it is resulted in black sludge without formation of any product, even when I was trying to use Cs2CO3 as a base. On the other hand, I was able to substitute 6-chlorine with amine with good yield (~70%, [MK147-1](https://au-mynotebook.labarchives.com/share/Marat%2520Korsik/NDI5LjB8MjkzNzAvMzMwL1RyZWVOb2RlLzI2MjkyNTEyNDF8MTA4OS4w)), but I still need to verify that I made expected product.

[6-substituted compounds.zip](https://github.com/OpenSourceMalaria/Series4/files/2508931/6-substituted.compounds.zip)

**Comment by MFernflower** [comment]
@maratsydney Perhaps try coupling the alcohol to the core using pyridine? 

I would be super curious as to how the analogue of the indole hit preforms:
![t6tf66y7](https://user-images.githubusercontent.com/3164942/47412490-e501ac80-d739-11e8-9dd9-c156b2b258ff.png)
Where X = H or F

**Comment by MFernflower** [comment]
Sorry about the accidental closure of this issue - my galaxy s8 has been acting up recently

**Comment by MFernflower** [comment]
@maratsydney I feel these compounds might be worth trying to produce:
![6sub](https://user-images.githubusercontent.com/3164942/47937763-26bbf100-deb8-11e8-9213-c0c9476a14d4.png)

**Comment by MFernflower** [comment]
Have you tried an organic base like tri or di ethylamine at low temperature
@maratsydney?

**Comment by MFernflower** [comment]
My bad I forgot that you already tried et3n

**Comment by MFernflower** [comment]
Please excuse my poor memory but was 6 amine tested and found to be nonpotent? @maratsydney Are there still plans to make 6 ethers or have they been overshadowed by other tasks at hand?

**Comment by maratsydney** [comment]
@MFernflower after looking deeper into spectral data of that MK147-1 6-amino compound it is most likely has other structure, currently sample was submitted for X-ray analysis.

**Comment by MFernflower** [comment]
Marat, when you have time can you spin that off into a new GHI? @maratsydney

**Comment by mattodd** [comment]
@maratsydney could you link to one or more ELN entries that describe the amine synthesis? It's an interesting work in progress.

The ether compound is, from my point of view, still of great value here, even though the synthesis is significantly more difficult (@maratsydney again it'd help if you could update this issue on where you're up to with some text and links). As it stands, the [data we have inherited for this compound](https://github.com/OpenSourceMalaria/Series4/wiki/Modification-of-Pyrazine-Substitution-Pattern) needs some investigation before we can explain why this line of inquiry was not pursued further. The original data would not strongly support your suggestion @MFernflower of trying the benzyl ether analogous to MMV670946.

**Comment by maratsydney** [comment]
All spectroscopy data for the MK147-1 including 2D-NMR  and HRMS can be found here [MK147-1](https://au-mynotebook.labarchives.com/share/Marat%2520Korsik/NDI5LjB8MjkzNzAvMzMwL1RyZWVOb2RlLzI2MjkyNTEyNDF8MTA4OS4w).  Sample submitted for the X-ray turned out to be poor quality for detailed analysis, so currently I am working on making a better crystals. 
![mk147-1_unknown](https://user-images.githubusercontent.com/29498087/51296218-325a9500-1a6f-11e9-868e-56997535b004.png)

Meanwhile I tried to couple MK145-1 with secondary amine [MK156-1](https://au-mynotebook.labarchives.com/share/Marat%2520Korsik/NDU4LjkwMDAwMDAwMDAwMDAzfDI5MzcwLzM1My9UcmVlTm9kZS85MzMwMTA5ODl8MTE2NC44OTk5OTk5OTk5OTk5) and received another unknown product, some preliminary data as well can be found in ELN: 
![mk156-1 unknown](https://user-images.githubusercontent.com/29498087/51295931-e3603000-1a6d-11e9-883c-b7bdfa0a40f7.png)
I will keep you updated as more information will be in my hands.

As per work towards making MK146-1 for the re-evaluation it is still in progress I tried to attach alcohol before cyclisation ([MK157-1](https://au-mynotebook.labarchives.com/share/Marat%2520Korsik/NDYxLjV8MjkzNzAvMzU1L1RyZWVOb2RlLzIwNTA3NTY1MjF8MTE3MS41)) but it didn't work out as analysis by NMR and TLC doesn't reveal new products being formed:
![mk157-1 nogo](https://user-images.githubusercontent.com/29498087/51295304-1ead2f80-1a6b-11e9-907f-9d5042a6d375.png)
As a next step I will try to install alcohol on pyrazine before hydrazine:
![alternative route](https://user-images.githubusercontent.com/29498087/51295482-f540d380-1a6b-11e9-9bca-2de4957df34d.png)

**Comment by MFernflower** [comment]
@maratsydney is it possible some sort of impurity in the silica reacted
with the chlorotriazolopyrazine functionality?

**Comment by MFernflower** [comment]
@maratsydney Mass spec should show if the product contains any silicon atoms in it

**Comment by maratsydney** [comment]
@MFernflower mass spec for MK147-1 showed expected mass so it must be some kind of isomer, as well silica is not the most reactive material to expect any product from there.

**Comment by MFernflower** [comment]
@maratsydney Have you attempted lowering the temperature of the hydrazine reaction? Stoppered RBF @ RT for like 8 hours maybe?

**Comment by MFernflower** [comment]
@maratsydney @mattodd The 3-(4-methoxyphenyl)-5-(phenethyl-1H-imidazol-2-yl)-4H-1,2,4-triazole side product you isolated interests me - perhaps we can send for malaria assay and/or Wendy for mycetoma assay?

**Comment by MFernflower** [comment]
@mattodd The 3-(4-methoxyphenyl)-5-(phenethyl-1H-imidazol-2-yl)-4H-1,2,4-triazole side product @maratsydney isolated interests me - perhaps we can send for malaria assay and/or Wendy for mycetoma assay? How many mg you have on hand?

**Comment by MFernflower** [comment]
@maratsydney I'm also somewhat curious as to if you run the same reaction again but at reflux instead of 80 C what percentage of the products would be the imidazole-triazole

**Comment by maratsydney** [comment]
@MFernflower actually we already tested it against malaria parasite (OSM-S-631). I submitted it without knowing the structure, now I have changed relevant data in master list and on ScienceCloud.  I still have about 20 mg of it. 

As for trying to run reaction at higher temperature or other alternations to increase yield of this interesting product: I will probably try it later as main focus for me currently on writing results of my tele-substitution studied into a paper.

**Comment by MFernflower** [comment]
Fascinating stuff!!! What was the potency of the compound vs malaria? Would it be possible to send 8 mg worth to @wwjvdsande ?? The structure kinda makes me think of something like this https://en.wikipedia.org/wiki/Netropsin

**Comment by maratsydney** [comment]
@MFernflower it came back as non potent. (IC50> 25uM).  I  would be happy to send it, just need to discuss with @mattodd if it worth it.

**Comment by MFernflower** [comment]
Sounds great

**Comment by MFernflower** [comment]
Sounds good - I know this is of low priority but do you have other substituted phenyl ethyl amines on hand? I'd be curious to see if different ones effect the yield of that weird degradation product

**Comment by MFernflower** [comment]
2-Fluorophenethylamine is cheap and not a controlled substance

**Comment by MFernflower** [comment]
@maratsydney Since you are making bulky derivatives of the s4 drugs to explore the chemistry I assume they will be assayed at some point vs the parasite?

Would it be possible to make a para trimethylsilyl compound?

**Comment by maratsydney** [comment]
@MFernflower Sure I will send them for evaluation at some point later. For trimethylsilyl compound we would have to wait for results, and if potency of compounds will be very good then we would explore it more.
